app/build.gradle

```diff
@@ -47,33 +47,25 @@ android {
 
     buildTypes {
         debug {
-            if (localPropertiesFile.exists()) {
-                def localProperties = new Properties()
-                localProperties.load(new FileInputStream(localPropertiesFile))
-
-                if(localProperties.getProperty('youtubeApiKey') != null && !localProperties.getProperty('youtubeApiKey').isEmpty()){
-                    resValue("string", "youtubeApiKey", localProperties['youtubeApiKey'])
-                }
-                else{
-                    resValue("string", "youtubeApiKey", "test")
-                }
-
-                if (localProperties.getProperty('spotifyClientId') != null && !localProperties.getProperty('spotifyClientId').isEmpty()){
-                    resValue("string", "spotifyClientId", localProperties['spotifyClientId'])
-                }
-                else{
-                    resValue("string", "spotifyClientId", "test")
-                }
-            }
-            else{
-                resValue("string", "youtubeApiKey", "test")
-                resValue("string", "spotifyClientId", "test")
-            }
-            resValue("string", "sentryDsn", "")
+            addLocalProperties(debug, localPropertiesFile)
+
+            buildConfigField "String", "API_BASE_URL", '"https://api.listenbrainz.org/1/"'
+            buildConfigField "String", "WEBSITE_BASE_URL", '"https://listenbrainz.org/"'
 
             applicationIdSuffix ".debug"
             versionNameSuffix ".debug"
         }
+        beta {
+            debuggable true
+            addLocalProperties(beta, localPropertiesFile)
+
+            buildConfigField "String", "API_BASE_URL", '"https://beta-api.listenbrainz.org/1/"'
```

Comment:
Why are we hardcoding fields in the gradle file? We should just pass a common field and then use that in the code to separate the values. The gradle file is already bloated and we shouldn't do this here at all.

Reply:
We could move it to gradle.properties if it were to be changed but because this will not change, we can keep it here for now.